test-apps/imodel-transformer/src/test/Transformer.test.ts

```diff
@@ -84,4 +85,52 @@ describe("imodel-transformer", () => {
     assert.isAtLeast(count(sourceDb, PhysicalPartition.classFullName), 2);
     targetDb.close();
   });
+
+  it("should exclude categories", async () => {
+    const targetDbFileName = initOutputFile("CompatibilityTestSeed-CategoryExcluded.bim");
+    const targetDb = SnapshotDb.createEmpty(targetDbFileName, {
+      rootSubject: { name: `${sourceDb.rootSubject.name}-CategoryExcluded` },
+      ecefLocation: sourceDb.ecefLocation,
+    });
+
+    const testSpatialCategory = {
+      name: "TestSpatialCategory",
+      id: "0x14",
+      code: new Code({
+        scope: "0x10",
+        spec: "0x16",
+        value: "TestSpatialCategory",
+      }),
+    };
+
+    await Transformer.transformAll(new BackendRequestContext(), sourceDb, targetDb, { excludeCategories: [testSpatialCategory.name] });
+
+    async function getElementsInTestCategory(db: IModelDb) {
+      const result: GeometricElementProps[] = [];
+      // do two queries because querying abstract GeometricElement won't contain the category
+      for (const type of [GeometricElement2d.classFullName, GeometricElement3d.classFullName]) {
+        for await (const elem of db.query(
+          `SELECT * FROM ${type} e JOIN bis.Category c ON e.category.id=c.ECInstanceId WHERE c.CodeValue=:category`,

```

Comment:
Could just SELECT ECInstanceId rather than * and return an Id64Set or Id64Array or SELECT COUNT(*) and return a number.

Reply:
I'll switch to COUNT, originally I was going to go through and make sure each element is not in the target but I think is is enough